apps/demo/vite.config.js

```diff
@@ -1,10 +1,19 @@
 import react from '@vitejs/plugin-react';
+// import wasmLoader from 'esbuild-plugin-wasm';
 import { defineConfig } from 'vite';
 import eslintPlugin from 'vite-plugin-eslint';
 
 export default defineConfig({
   server: { open: true },
   preview: { open: true },
+  esbuild: {
+    target: 'es2020',
+    sourcemap: true,
+  },
+  build: {
+    target: 'es2020',
+    sourcemap: true,
+  },
```

Comment:
Some of this was me getting familiar with the build environment you are using.  I was under the impression one of those sections was used for the preview mode, and the other during build of production assets.

You are correct that this is related to BigInt support.  BigInt is being used for pointers being passed to the WASM code.  These could be wrapped into plain numbers since currently wasm in browsers only supports 32-bit addresses, and 32-bit ints fit into a plain javascript number, though that would require a bit of work in the h5wasm code.  BigInt is pretty well supported across browsers and platforms at this time.

Lazy loading sounds nice - I am new to React and Vite so I wouldn't know how to set that up.

Reply:
We can take a look at that, no worries ;)